src/dlgTriggerEditor.cpp

```diff
@@ -125,68 +119,56 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     setStatusBar( statusBar );
     statusBar->show();
     mIsGrabKey = false;
-    QVBoxLayout * pVB1 = new QVBoxLayout(mainArea);
+
+    // Replacement code to use single splitter for all right widgets:
+    mpSystemMessageArea = new dlgSystemMessageArea(splitter_rightTopBottom);
+    mpTriggersMainArea = new dlgTriggersMainArea(splitter_rightTopBottom);
+    mpTimersMainArea = new dlgTimersMainArea(splitter_rightTopBottom);
+    mpAliasMainArea = new dlgAliasMainArea(splitter_rightTopBottom);
+    mpActionsMainArea = new dlgActionMainArea(splitter_rightTopBottom);
+    mpKeysMainArea = new dlgKeysMainArea(splitter_rightTopBottom);
+    mpVarsMainArea = new dlgVarsMainArea(splitter_rightTopBottom);
+    mpScriptsMainArea = new dlgScriptsMainArea(splitter_rightTopBottom);
+    mpSourceEditorArea = new dlgSourceEditorArea(splitter_rightTopBottom);
+    mpErrorConsole = new TConsole(mpHost,false,splitter_rightTopBottom);
+    mpErrorConsole->setMinimumHeight( 200 );
+
+    //Prevent any region being "squashed down" i.e. taken below its minimum size
+    //so that it "disappears"
+    splitter_rightTopBottom->setChildrenCollapsible(false);
+    splitter_main->setChildrenCollapsible(false);
+
+    splitter_rightTopBottom->addWidget(mpSystemMessageArea);
+    splitter_rightTopBottom->addWidget(mpTriggersMainArea);
+    splitter_rightTopBottom->addWidget(mpTimersMainArea);
+    splitter_rightTopBottom->addWidget(mpAliasMainArea);
+    splitter_rightTopBottom->addWidget(mpActionsMainArea);
+    splitter_rightTopBottom->addWidget(mpKeysMainArea);
+    splitter_rightTopBottom->addWidget(mpVarsMainArea);
+    splitter_rightTopBottom->addWidget(mpScriptsMainArea);
```

Comment:
That is why I allowed for some expansion of the Registered Event Handlers `QListWidget` - I guess it needs more - actually, since it says "Script name:" is that the prior code or the version in this PR?